app/views/coronavirus_local_restrictions/show.html.erb

```diff
@@ -1,6 +1,12 @@
 <%
+  error_key = if @search&.empty?
```

Comment:
It doesn't ever seem particularly intuitive passing locals through the Rails controller rather than using instance variables. I think it's fine to use instance variables in the main view for an action. They just shouldn't be used beyond that as they then creep into global variables.

Ideally, Rails would push an intuitive way for locals to be defined in controllers as the default.